src/main/resources/assets/notenoughanimations/lang/tr_tr.json

```diff
@@ -57,15 +57,25 @@
     "text.nea.enable.freezingAnimation": "Donma Animasyonu",
     "text.nea.enable.freezingAnimation.tooltip": "Donma için bir animasyon ekler",
     "text.nea.enable.huggingAnimation": "Basit Sarılma Animasyonu",
-    "text.nea.enable.huggingAniFmation.tooltip": "Sarılmak için bir animasyon ekler. Başka bir oyuncuya 'sarılmak' için eğilirken ona bakın",
+    "text.nea.enable.huggingAnimation.tooltip": "Sarılmak için bir animasyon ekler. Başka bir oyuncuya 'sarılmak' için eğilirken ona bakın",
     "text.nea.enable.narutoRunning": "❝Naruto Koşusu❞",
     "text.nea.enable.narutoRunning.tooltip": "Naruto tarzında koşmak için bir animasyon ekler (easter egg)",
-    "text.nea.enable.enableInWorldBookRendering": "[Yapım Aşamasında]Dünya İçi Kitaplars",
+    "text.nea.enable.enableInWorldBookRendering": "[Yapım Aşamasında]Dünya İçi Kitaplar",
     "text.nea.enable.enableInWorldBookRendering.tooltip": "Dünyadaki kitapları metinleriyle birlikte işler. Bu hala yapım aşamasındadır ve düzgün çalışmayabilir",
     "text.nea.disableLegSmoothing": "Bacak Yumuşatmayı Devre Dışı Bırak",
     "text.nea.disableLegSmoothing.tooltip": "Bacakların yumuşatılmasını devre dışı bırakır",
     "text.nea.enable.bowAnimation": "Yay Animasyonu",
     "text.nea.enable.bowAnimation.tooltip": "Kullanılacak yay animasyonu tipi",
     "text.nea.enable.bowAnimation.VANILLA": "Vanilla",
     "text.nea.enable.bowAnimation.CUSTOM_V1": "Özel V1"
```

Comment:
There is a , missing here, thats why the build crashed.

Reply:
Sorry about that. Will be more careful next time.

Reply:
It's not a problem, spotless thankfully catches issues like this :D